code/modules/surgery/organs_internal.dm

```diff
@@ -156,6 +156,14 @@
 					"\blue You treat damage to [target]'s [I.name] with [tool_name]." )
 					I.damage = 0
 
+		if(target_zone == "head")
```

Comment:
thing is a lot of these are completely decoupled from eyes--for instance, NEARSIGHTED and BLIND actually have...very little to do with eyes; blind and blurry, as well---and there's definitely not an "ears" organ. *shrug* Dunno.